game-core/src/main/java/games/strategy/triplea/ui/screen/Tile.java

```diff
@@ -27,9 +26,9 @@
 public class Tile {
   public static final LockUtil LOCK_UTIL = LockUtil.INSTANCE;
 
-  // allow the gc to implement memory management
-  private SoftReference<Image> imageRef;
```

Comment:
Reasoning for this change? Concerned this could impact memory usage. Did you run any tests?

Reply:
Memory usage should not be a concern for images that should more or less constantly be displayed on the screen (the previous code just re-loaded the image in case the reference was lost). So the only thing that could happen is a potential memory leak so that the references are never freed.
I did some manual testing by opening and closing the game window multiple times. Memory always seemed to go back to its initial value after a while (after gc) so we should be fine.